android/src/app/organicmaps/bookmarks/Holders.java

```diff
@@ -275,6 +285,15 @@ void setSize(@PluralsRes int phrase, int size)
       mSize.setText(mSize.getResources().getQuantityString(phrase, size, size));
     }
 
+    void setDetailedSize(int tracks, int bookmarks)
```

Comment:
It looks like a duplicate of the code above.

Reply:
I see, these are two view-holders, one for bookmarks, one for collections. Do you think it is better to add one more object to avoid duplication?